Packs/SendGrid/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "SendGrid",
+    "description": "Send, manage your emails.",
```

Comment:
-> "SendGrid provides a cloud-based service that assists businesses with email delivery. It allows companies to track email opens, unsubscribes, bounces, and spam reports. Our SendGrid pack utilize these SendGrid use cases to help you send and manage your emails."